src/components/ReportRange/index.js

```diff
@@ -30,6 +31,10 @@ class ReportRange extends Component<ReportRangeType> {
       startOfWeek(new Date(), { weekStartsOn: 1 }),
       endOfWeek(new Date(), { weekStartsOn: 1 }),
     );
+    this.weekToDate = () => this.props.updateDateRange(
```

Comment:
Can you rename this to `onWeekToDate ` to be consistent with the other functions?